firebase-ml-modeldownloader/src/main/java/com/google/firebase/ml/modeldownloader/FirebaseModelDownloader.java

```diff
@@ -120,6 +120,16 @@ public static FirebaseModelDownloader getInstance(@NonNull FirebaseApp app) {
    *       trigger new download, task only completes when download finishes
    * </ul>
    *
+   * Most common exceptions include:
+   *
+   * <ul>
+   *   <li>{@link FirebaseMlException#NOT_FOUND}: No model found with the given name.
+   *   <li>{@link FirebaseMlException#NOT_ENOUGH_SPACE}: Not enough space on device to download
+   *       model.
+   *   <li>{@link FirebaseMlException#DOWNLOAD_URL_EXPIRED}: Url used to fetch model expired before
+   *       model download completed. (Retry suggested)
```

Comment:
This one should never happen, right? or at least should not be common...
What about network errors? Do those happen?